react-design/my-app/src/Banner.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Banner({ item }) {
+  return <div style={{ fontSize: '40px', marginBottom: '2rem' }}>{item}</div>;
```

Comment:
Try to use consistent units on font-adjacent properties (ex: `font-size`, `padding`, `border`, `margin`, etc.)

Reply:
Made adjustments, thanks for flagging!